htdocs/js/jquery.imageshrink.js

```diff
@@ -18,19 +25,19 @@ jQuery(function($) {
     // Only shrink casual pics; leave artisanal HTML alone. Can't predict
     // everything, but 99% of the time that's a <div style="..."> or a table.
     function protectTheCodes(container) {
-        // Expanded cut tags have style='display: block', but aren't decorations.
-        // Feeds (.journal-type-Y) always make a mess, so no mercy.
-        var exemptImages = container.querySelectorAll(
-            '.journal-type-P .entry-content div[style]:not(.cuttag-open) img, ' +
-            '.journal-type-P .entry-content table img, ' +
-            '.journal-type-C .entry-content div[style]:not(.cuttag-open) img, ' +
-            '.journal-type-C .entry-content table img, ' +
-            '.comment-content div[style] img, ' +
-            '.comment-content table img'
-        );
+        let queryStrings = imgHolders.map(function(holderName) {
+            // Expanded cut tags have style='display: block', but aren't decorations.
+            return `${holderName} div[style]:not(.cuttag-open) img, ${holderName} table img`;
+        });
+
+        var exemptImages = container.querySelectorAll( queryStrings.join(', ') );
         for (var i = 0; i < exemptImages.length; i++) {
             exemptImages[i].classList.add('imageshrink-exempt');
         }
+
+        // Feeds (.journal-type-Y) always make a mess, so no mercy.
+        container.querySelectorAll('.journal-type-Y img').forEach( function(img) { img.classList.remove('imageshrink-exempt')})
```

Comment:
Well.................... okay. 😂 I guess it's better than the other things I might suggest.